### DISCUSSION TITLE
milli-sat payments?

### DISCUSSION THREAD

**Discussion by sw1mclub** [lightningnetwork/lnd#5670] [Q&A] [answered]
I'm trying to make payments in the milli-sat range, but it seems that a route can't be found to actually perform the payment. 

I looked at the query route command and it seems to only accept an amt in sats which makes me think maybe this just hasn't been built out yet?
https://github.com/lightningnetwork/lnd/blob/f7384a09e610411967f5741b9f256abb23f40f74/cmd/lncli/commands.go#L1807

I'm new to this project so maybe this is an intentional design decision.

Example payment request: lnbcrt120p1psjwgnhpp5wc2t58pr47snxjmekr546nkd4d8eu7tj79uuda9f2n2azwjp22vqdq6dpc85um9wfmxjcm9w3jhxarfvscqzpgxqyz5vqsp5y2dq0ujcw5s9c37l2zdud74rczfa6egpp8hrutc747smxlgxrtsq9qyyssq6wqelx2deu2uqxpj8pvjh2vdcjyz92c47khj77gptkru6cgvtv3rg9sts9j3x0m3cgldjjzxtukwh2yhlm3chxevwj7h7snel38zy4cqec45zf

Example Error:
Failed to find route for payment 0431ae36cde843cf3f8b5b8b3589086417c60b27ee2ff8a315f6b57d89a31e28: unable to find a path to destination

**Comment by guggero** [comment, marked as answer]
Nodes can define a minimum amount they are willing to forward. Perhaps there just isn't a route that allows for small payments.
Also make sure you set a non-zero `fee_limit`, otherwise only "free" routes will be searched, which might not exist in your case.

**Reply by sw1mclub** [reply]
Thanks! Opening a channel with option `openchannel --min_htlc_msat 1` does the trick
